server/lib/chat/chat-service.ts

```diff
@@ -158,12 +176,16 @@ export default class ChatService {
       limit,
       beforeTime && beforeTime > -1 ? new Date(beforeTime) : undefined,
     )
-    return messages.map(m => ({
-      id: m.msgId,
-      user: m.userName,
-      sent: +m.sent,
-      data: m.data,
-    }))
+    return messages.map(m => {
```

Comment:
nit: you can write this more succinctly by passing a generic parameter to `map`, i.e.

```ts
return messages.map<ChatMessage>(m => ({
  id: m.msgId,
  user: m.userName,
  sent: Number(m.sent),
  data: m.data,
}))
```

(applies to other places in this file too)